source/components/List/styles.scss

```diff
@@ -0,0 +1 @@
+@import "~design-system/dist/css/styles.css";
```

Comment:
I needed the design system styles so the example code from the README would work. Is this the proper way to have that added?

Reply:
It's not only needed for the examples but also it's going to be compiled to output CSS and it needs to be included in the app that's using `react-common`

It should be:
```
@import "~design-system/dist/scss/wds-variables/index.scss";
@import "~design-system/dist/scss/wds-mixins/index.scss";
@import "~design-system/dist/scss/wds-components/_lists.scss";
```

I've fixed that for you in c5b1274

Reply:
Awesome, thank you very much @vforge!